docs/4.3/admin-guide.md

```diff
@@ -2259,11 +2261,62 @@ When upgrading multiple clusters:
 
 ## Backing Up Teleport
 
+When planning a backup of Teleport, it's important to know what is where and the
+importance of each component. Teleport's Proxies and Nodes are stateless, and thus
+only `teleport.yaml` should be backed up.
+
+The Auth server is Teleport's brains, and depending on the backend should be backed up
+in a regular cadance.
```

Comment:
```suggestion
regularly.
```

(can't decide on "at a regular cadence" versus "in a regular cadence", so let's just change it to "regularly"?)